docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: "3"
+services:
+  mongo-container:
+    image: mongo:latest
+    container_name: "mongo-container"
```

Comment:
This doesn't hurt anything, however this does prevent more than one instance of banter-board-backend from running at the same time since docker container names need to be unique. 

See "container_name" docs
https://docs.docker.com/compose/compose-file/#links

This is more just a piece of trivia than something that will actually matter.